walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -0,0 +1,458 @@
+// tag::master-1[]
+
+:walkthrough: No code API development and SaaS integration
+:fuse-version: 7.2
+:3scale-name: Red Hat 3scale API Management Platform
+:3scale-version: 7.2
+:fuse-url: https://eval.apps.city.openshiftworkshop.com/
+:3scale-url: https://eval.apps.city.openshiftworkshop.com/
+:route: https://wt2-{user-sanitized-username}-3scale.{openshift-app-host}
+
+[id='no-code-container-native-api-development']
+
+= {walkthrough}
+
+This walkthrough demonstrates how to implement a container-native OpenAPI RESTful API without coding.
+
+Container-native applications::
+Applications that are designed to run in a containerised environment.
+
+// This is taken right from https://github.com/OAI/OpenAPI-Specification
+OpenAPI Specification::
+A standard, programming language-agnostic interface description for REST APIs, which allows both humans and computers to discover and understand the capabilities of a service without requiring access to source code, additional documentation, or inspection of network traffic. When properly defined via OpenAPI, a consumer can understand and interact with the remote service with a minimal amount of implementation logic.
+
+In this walkthrough you follow a design-first approach, based on the OpenAPI specification, to define a RESTful API that accepts a name parameter and echoes it back in the response body. 
+You then use the resulting Open API definition to generate the code for your API and deploy it in a containerised environment.
+
+Following a design first approach to API development is self-documenting, and provides a clear API definition for partners and end-users of your API.
+It also allows you to identify important design decisions prior to coding and deployment.
+
+[type=walkthroughResource,serviceName=fuse]
+.Fuse Online
+****
+* link:{fuse-url}[Console, window="_blank"]
+****
+
+[type=walkthroughResource,serviceName=openshift]
+.Red Hat OpenShift
+****
+* link:{openshift-host}/console[Console, window="_blank"]
+****
+
+[type=walkthroughResource,serviceName=3scale]
+.3Scale
+****
+* link:{api-management-url}[Console, window="_blank"]
+* link:https://developers.redhat.com/products/3scale/overview/[3Scale Overview, window="_blank"]
+* link:https://www.3scale.net[3Scale Website, window="_blank"]
+****
+
+
+[time=10]
+== Creating a Slack Connection in Fuse Online
+
+In Fuse Online a *Connection* holds the information required to connect to a service.
+For example, a Slack *Connection* in Fuse Online requires an *API Token* and *Slack URL* to function. 
+
+=== Creating a Slack Workspace
+
+.Prerequisite
+You already have an account with link:https://slack.com/[Slack].
+
+.Procedure
+. Navigate to the link:https://slack.com/create[Create a new workspace, window="_blank"] page.
+
+. Enter your email address and click *Next*.
+
```

Comment:
If you have 2FA enabled on your slack account, you will have to enter the 2FA code after clicking *Next*.  It won't go straight to "What's the name of your company or team?"